frontend/src/containers/RecipePage/SpecsBar/DifficultySpec/DifficultySpec.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import './DifficultySpec.css';
+
+export default class DifficultySpec extends Component {
+  constructor() {
+    super();
+    this.state = {
+      boxColorTwo: '#eeeeee',
+      boxColorThree: '#eeeeee',
+      difficulty: '',
+    };
+  }
+
+  componentDidMount() {
+    const difficultyOptions = ['EASY', 'MEDIUM', 'HARD'];
+    const probabilityCheck = Math.random();
+
+    if (probabilityCheck < (1 / 3)) {
+      this.setState({ difficulty: difficultyOptions[0] });
+    } else if (probabilityCheck < (2 / 3)) {
+      this.setState({ difficulty: difficultyOptions[1], boxColorTwo: '#55ffff' });
+    } else {
+      this.setState({ difficulty: difficultyOptions[2], boxColorTwo: '#55ffff', boxColorThree: '#55ffff' });
+    }
```

Comment:
This is pretty neat, but I think just employing icons might be a more productive approach?

Reply:
Hey Brian, We didn't make the icons we just took them from the RecipeBase container for consistency.

Reply:
Ahh, yeah. The `RecipeBase` is definitely old, unused code. If you want to take a look at the `RecipeCard` component, that's what we're actually using right now.

Reply:
Gotcha, we corrected all icons to be from website. Should be ready for another code review!